custom_components/waste_collection_schedule/waste_collection_schedule/source/fkf_bo_hu.py

```diff
@@ -0,0 +1,86 @@
+import json
+import requests
+import datetime
+
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection
+
+TITLE = "FKF Budaörs"
+DESCRIPTION = "Source script for www.fkf.hu"
+URL = "https://www.fkf.hu"
+COUNTRY = "hu"
+TEST_CASES = {
+    "Success_Test_1": {"street": "Templom tér"},    # Should Succeed
+    "Success_Test_2": {"street": "Völgy utca"},     # Should Succeed
+    "Fail_Test_1": {"street": "Zombri utca"},       # Should Fail
+    "Fail_Test_2": {"street": "Pici utca"}          # Should Fail
```

Comment:
could you please remove or comment out failing tests as they suggest something went wrong. We currently do not have a good method to add "Fail Tests". Maybe something we could look into in the future.

Reply:
Ok, I removed the failing tests and added a third successful test